docs/content/doc/usage/https-support.md

```diff
@@ -27,8 +27,8 @@ To use Gitea's built-in HTTPS support, you must change your `app.ini` file:
 PROTOCOL  = https
 ROOT_URL  = https://git.example.com:3000/
 HTTP_PORT = 3000
-CERT_FILE = cert.pem
-KEY_FILE  = key.pem
+CERT_FILE = /etc/gitea/cert.pem
+KEY_FILE  = /etc/gitea/key.pem
```

Comment:
it should probably be better placed under `/etc/ssl/private/...`

Reply:
But the `git` user won't have permissions to access that folder, or will it?

Reply:
well I guess it depends on the permission you will give to the files but it's not entirely clear to me as I never managed to get gitea working with the certificate I have in the letsencrypt default dir:
/etc/letsencrypt/live/mysite.com/

Reply:
Depending on Linux distro, you have to add git user to ssl group to have such access

Reply:
I wouldn't add git to the ssl group. I don't think that's a good practice from the security point of view as Gitea is connected to internet and doesn't switch to a lesser able user, like Apache for example does. `/etc/ssl/private` may contain other SSL certificates, perhaps more sensitive than Gitea's. We need Gitea to read the private key for its certificate, that's unavoidable, but having permissions to read *any* private keys...
I'd stick to `/etc/gitea` for that.